app/controllers/users_controller.rb

```diff
@@ -64,4 +67,20 @@ def should_display_user?(user)
   def get_user
     @user = User.friendly.find(params[:id])
   end
+
+  def count_timezone_degrees
+    unless params[:timezone_select].blank?
```

Comment:
do we need line 72? 

looks like blank values are already guarded against in .filter